### DISCUSSION TITLE
The option to start Windows Terminal has disappeared from the right-click menu.

### DISCUSSION THREAD

**Discussion by Cufoon** [microsoft/terminal#18211] [Q&A]
I have tried various methods, but reinstalling doesn't solve the problem, and I can't find the relevant option in the registry.

**Comment by zadjii-msft** [comment]
What version of Windows are you using? 
What version of the Terminal?
Is it missing from the "Show more options" menu?

**Comment by zadjii-msft** [comment]
Looks like you also filed #18210, so I'll just close this discussion as a dupe of that thread. Thanks!
